broker-plugin/plugin/src/main/resources/bootstrap.xml

```diff
@@ -34,6 +34,9 @@
        <app url="console" war="console.war"/>
    </web>
 
+   <web bind="http://0.0.0.0:8080" path="web">
```

Comment:
How does the path element `web` work?   

This doesn't appear to match what is advertised via the deployment/statefulset broker annotations?

         prometheus.io/path: /metrics
          prometheus.io/port: '8080'
          prometheus.io/scrape: 'true'

prometheus.io/path

Reply:
Its actually the path in the filesystem for looking up the .war file (which I discovered eventually :D)

Reply:
Ah..  that kind of path!   So I guess this means the plugin just knows to bind to /metrics by default?

Reply:
Thats the 'url' field in the app element. I bound it to / , but I can bind it to /metrics instead.